src/plugins/intel_cpu/src/nodes/def_conv.cpp

```diff
@@ -1155,10 +1212,20 @@ void DeformableConvolution::prepareParams() {
     sampledCoordsVector.resize(MB * DG * KH * KW * OH * OW * sampledPointsPerPixel);
     interpWeightsVector.resize(MB * DG * KH * KW * OH * OW * sampledPointsPerPixel);
 
+    execPtr = nullptr;
+
     if (enforceRef) {
-        execPtr = std::make_shared<DefConvRefExecutor>(defConvAttr, descVector);
+        execPtr = std::make_shared<DefConvRefExecutor>(key.defConvAttr, key.descVector);
```

Comment:
Base constructor also have lots computations. Why don't we cache ref version as well? We can add `enforceRef` to the key.

Reply:
Ref. caching added. `enforceref` is implicitly contained in `DefConvKey` by `implType`.